middleware/grpc_logging.go

```diff
@@ -24,11 +25,21 @@ const (
 	gRPC = "gRPC"
 )
 
+var (
+	DoNotLogErrorHeader = http.CanonicalHeaderKey("X-DoNotLogError")
+)
```

Comment:
I think this belongs to httpgrpc package, where server is using it. `grpc_logging.go` has nothing to do with HTTP.

Reply:
I have moved this header to `httpgrpc/server.go`.